optimum/exporters/openvino/__main__.py

```diff
@@ -317,9 +317,7 @@ class StoreAttr(object):
                 )
             model.config.pad_token_id = pad_token_id
 
-    if "stable-diffusion" in task:
```

Comment:
to ensure compatibility with optimum following changes from https://github.com/huggingface/optimum/pull/1947 we need to update things like  https://github.com/huggingface/optimum-intel/blob/cb2f2eca18da8fcddba3ac6544da7a7a07d57f51/optimum/exporters/openvino/__main__.py#L197-L199 and https://github.com/huggingface/optimum-intel/blob/cb2f2eca18da8fcddba3ac6544da7a7a07d57f51/optimum/commands/export/openvino.py#L230 as well